ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/MacroRegistryImpl.java

```diff
@@ -39,9 +38,7 @@ public MacroRegistryImpl() {
     public void register(Set<Macro> macros) {
         for (Macro macro : macros) {
             final String name = macro.getName();
-            if (this.macros.containsKey(name)) {
-                Log.warn(MacroRegistryImpl.class, "Command macro '" + name + "' is already registered.");
```

Comment:
It's not a problem to revert the code.
And try imaging how much you will be happy when you develop your code and can not find your output because console has the output which informs you about nothing but disturbs.
Please think about other developers.